ledger/simulation/trace.go

```diff
@@ -70,11 +70,15 @@ type ResultEvalOverrides struct {
 	AllowEmptySignatures bool
 	MaxLogCalls          *uint64
 	MaxLogSize           *uint64
+	ExtraAppBudget       *uint64
 }
 
 // SimulateLogBytesLimit hardcode limit of how much bytes one can log per transaction during simulation (with AllowMoreLogging)
 const SimulateLogBytesLimit = uint64(65536)
 
+// SimulateMaxExtraBudget hardcode limit of how much extra budget one can add to one transaction group (which is group-size * logic-sig-budget)
+const SimulateMaxExtraBudget = uint64(20000 * 16)
```

Comment:
Since these globals are in the `simulation` package, Go convention would be to name them simply `MaxExtraBudget`, so that uses look like `simulation.MaxExtraBudget`.

`simulation.SimulateMaxExtraBudget` reads very repetitively.

Reply:
right, I also changed to `LogBytesLimit` above. See [review comments response 1](https://github.com/algorand/go-algorand/pull/5354/commits/1069eeed4dd0a4570bf6644e0cf6cd379d59d43f)